bin/martinize2

```diff
@@ -623,57 +534,85 @@ def entry():
         "-go",
         action="store_true",
         default=False,
-        help="Apply the Martini Go model as structural bias.",
-    )
-    go_group.add_argument(
-        "-govs-moltype",
-        dest="govs_moltype",
-        default="molecule_0",
-        help=("Set the name of the molecule when using " "Virtual Sites GoMartini."),
-    )
-    go_group.add_argument(
-        "-govs-includes",
-        action="store_true",
-        default=False,
-        help="Write include statements to use Vitrual Site Go Martini.",
+        help="Apply the Martini Go model as structural bias and/or water bias.",
     )
     go_group.add_argument(
-        "-govs-map",
+        "-go-map",
         dest="go_map",
         required=False,
         type=Path,
         default=False,
-        help="Write include statements to use Vitrual Site Go Martini.",
+        help="Contact map to be used for the Martini Go model."
+             "Currently, only one format is supported. See docs.",
     )
     go_group.add_argument(
-        "-govs-low",
-        dest="go_low",
+        "-go-eps",
+        dest="go_eps",
         type=float,
-        default=0.3,
+        default=0.0,
+        help=("The strength of the Go model structural bias in kJ/mol."),
+    )
+    go_group.add_argument(
+        "-go-moltype",
+        dest="govs_moltype",
+        default="molecule_0",
         help=("Set the name of the molecule when using " "Virtual Sites GoMartini."),
     )
```

Comment:
As previously explained: 

Turns out this flag is very important! Currently, it is not possible to run the Go map for larger(ish) complexes. Thus, one common workflow routine consists of multiple martinize2 runs for different assemblies of the large complex. Each of these assemblies get's a Go bias potential. Then everything is combined together. Having the moltype flag allows the user to set the molecule name while looping over these. If removed it becomes super annoying because all prefixes of the Go-type have to be changed. So we very much want to keep this.

I think we might even want to allow setting of a molecule name prefix in general